demos/text_detection_demo/main.cpp

```diff
@@ -294,9 +294,9 @@ int main(int argc, char *argv[]) {
           std::cout << "text detection model inference (ms) (fps): "
                     << text_detection.time_elapsed() / text_detection.ncalls() << " "
                     << text_detection.ncalls() * 1000 / text_detection.time_elapsed() << std::endl;
-        if (std::fabs(text_detection_postproc_time) < std::numeric_limits<double>::epsilon()) {
-            throw std::logic_error("text_detection_postproc_time can't be equal to zero");
-        }
+        if (std::fabs(text_detection_postproc_time) < std::numeric_limits<double>::epsilon())
```

Comment:
@GalyaZalesskaya why did you remove curly braces?

Reply:
For the sameness with lines [above](https://github.com/GalyaZalesskaya/open_model_zoo/blob/da991e8cb0b82c714adad571291725c779dead0a/demos/text_detection_demo/main.cpp#L294) and [below](https://github.com/GalyaZalesskaya/open_model_zoo/blob/da991e8cb0b82c714adad571291725c779dead0a/demos/text_detection_demo/main.cpp#L300) where one std::cout operator is not bracketed